cluster/charts/rook-ceph-cluster/values.yaml

```diff
@@ -410,6 +410,7 @@ cephObjectStores:
     storageClass:
       enabled: true
       name: ceph-bucket
+      isDefault: false
```

Comment:
This should go under the `cephFilesystem.storageClass` instead of under the `cephObjectStores`, right?

Reply:
Sure, fixed! Sorry :)